nestjs-counter/apps/worker/src/exchange-rates-worker/exchange-rates-worker.providers.ts

```diff
@@ -0,0 +1,33 @@
+import { ActivitiesService } from '../activities/activities.service';
+import { DefaultLogger, LogEntry, Runtime, Worker } from '@temporalio/worker';
+import { taskQueue } from '@app/shared';
+
+export const exchangeRatesWorkerProviders = [
+  {
+    provide: 'EXCHANGE_RATES_WORKER',
+    inject: [ActivitiesService],
+    useFactory: async (activitiesService: ActivitiesService) => {
+      Runtime.install({
+        logger: new DefaultLogger('WARN', (entry: LogEntry) =>
+          console.log(`[${entry.level}]`, entry.message),
+        ),
+      });
```

Comment:
Why change the logger, especially don't love that we lose the timestamps here.
Also to follow the nestjs philosophy, `Runtime` should be a provider.